pkg/asset/machines/baremetal/hosts.go

```diff
@@ -103,6 +120,15 @@ func Hosts(config *types.InstallConfig, machines []machineapi.Machine) (*HostSet
 		}
 		newHost := createBaremetalHost(host, bmc)
 
+		if host.NetworkConfig != "" {
+			networkConfigSecret := createNetworkConfigSecret(host)
+			settings.NetworkConfigSecrets = append(settings.NetworkConfigSecrets, networkConfigSecret)
+			// newHost.Spec.PreprovisioningNetworkDataName = &corev1.SecretReference{
```

Comment:
Remove this commented out code, especially as it is removed in a latter commit in this PR.

Reply:
Done